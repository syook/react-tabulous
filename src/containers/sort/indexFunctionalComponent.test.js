import React from 'react';
import { configure, mount, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import SortProvider from './indexFunctionalComponent';

configure({ adapter: new Adapter() });

describe('SortProvider', () => {
  it('should render without crashing', () => {
    shallow(<SortProvider />);
  });

  it('should match snapshot', () => {
    const tree = shallow(<SortProvider />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
