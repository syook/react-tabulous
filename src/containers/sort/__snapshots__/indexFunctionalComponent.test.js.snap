// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortProvider should match snapshot 1`] = `
<ContextProvider
  value={
    Object {
      "columnName": null,
      "columnType": null,
      "count": undefined,
      "data": Array [],
      "direction": null,
      "handleSort": [Function],
      "rawData": undefined,
      "resetPagination": false,
    }
  }
/>
`;
